control/veda_video_validation.py

```diff
@@ -22,105 +22,96 @@
 logging.basicConfig(stream=sys.stdout, level=logging.INFO)
```

Comment:
Putting this here more for myself to remember, and also for anybody looking at this later:

This is a big diff, but all I'm doing here is turning Validation from a class to a method.

FYI @schenedx